### PR TITLE
AUTO: LIVE 3.6.2

### PR DESCRIPTION
**Pull request by ann0see**
@jamulussoftware/translators This pull request will **publish** the newly translated changes to jamulus.io. Please review the site locally. This PR will be merged soon.

### CODE REVIEW

File: _data/es/navigation.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ nav:
       subsubfolderitems:
         - page:          Windows
           url:           /wiki/Installation-for-Windows
-        - page:          Macintosh
+        - page:          macOS
           url:           /wiki/Installation-for-Macintosh
         - page:          Linux
           url:           /wiki/Installation-for-Linux
```
Comment (ann0see) [inline _data/es/navigation.yml:12]: @ignotus666 The link to the "Onboarding" page seems to be missing for the spanish translation.

**Comment by ann0see** [comment]
Broken links:

http://127.0.0.1:4000/de/wiki/Client-Troubleshooting
- (194:101) 'ASIO4All..' => http://127.0.0.1:4000/de/wiki/Installation-for-Windows#setting-up-asio4all (HTTP 200 but missing anchor)

http://127.0.0.1:4000/de/wiki/Command-Line-Options
- (353:29) 'Tipps & ..' => http://127.0.0.1:4000/de/wiki/Tips-Tricks-More#Using-ctrlmidich-for-MIDI-controllers (HTTP 200 but missing anchor)

http://127.0.0.1:4000/de/wiki/Hardware-Setup
- (237:22) 'client i..' => http://127.0.0.1:4000/de/wiki/Installation-for-Linux#configure-jack-with-qjackctl (HTTP 200 but missing anchor)

http://127.0.0.1:4000/de/wiki/Server-Linux
- (219:6) 'Get the ..' => http://127.0.0.1:4000/de/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)
- (219:92) 'dependen..' => http://127.0.0.1:4000/de/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)
- (399:248) 'More inf..' => http://127.0.0.1:4000/de/wiki/Compiling#the-headless-build-flag (HTTP 200 but missing anchor)

http://127.0.0.1:4000/de/wiki/Server-Rpi
- (203:7) 'Get the ..' => http://127.0.0.1:4000/de/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)                                                                            
- (203:93) 'dependen..' => http://127.0.0.1:4000/de/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)                                                                          
- (206:155) 'default ..' => http://127.0.0.1:4000/de/wiki/Installation-for-Linux#compile-this-bad-boy (HTTP 200 but missing anchor)                                                                         
                                                                                                      
http://127.0.0.1:4000/de/wiki/Server-Win-Mac                                                          
- (243:99) 'private ..' => http://127.0.0.1:4000/de/wiki/Choosing-a-Server-Type#3-central (HTTP 200 but missing anchor)                                                                                     
                                                                                                      
http://127.0.0.1:4000/es/wiki/Client-Troubleshooting                                                  
- (234:92) 'resoluci..' => http://127.0.0.1:4000/es/wiki/Server-Troubleshooting#nadie-se-puede-conectar-a-mi-servidor---pero-yo-me-puedo-conectar-localmente (HTTP 200 but missing anchor)                  
                                                                                                      
http://127.0.0.1:4000/es/wiki/Compiling
- (185:128) 'compilar..' => http://127.0.0.1:4000/es/wiki/Server-Linux#ejecutar-un-servidor-headless (HTTP 200 but missing anchor)                                                                          
- (256:175) 'servidor..' => http://127.0.0.1:4000/es/wiki/Server-Linux#ejecutar-un-servidor-headless (HTTP 200 but missing anchor)                                                                          

http://127.0.0.1:4000/es/wiki/Hardware-Setup
- (229:15) 'guía de ..' => http://127.0.0.1:4000/es/wiki/Installation-for-Linux#configure-jack-with-qjackctl (HTTP 200 but missing anchor)                                                                  

http://127.0.0.1:4000/es/wiki/Installation-for-Linux
- (287:248) 'servidor..' => http://127.0.0.1:4000/es/wiki/Server-Linux#ejecutar-un-servidor-headless (HTTP 200 but missing anchor)                                                                          

http://127.0.0.1:4000/es/wiki/Server-Linux
- (211:6) 'Get the ..' => http://127.0.0.1:4000/es/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)                                                                            
- (211:92) 'dependen..' => http://127.0.0.1:4000/es/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)                                                                          
- (329:18) 'recordin..' => http://127.0.0.1:4000/es/wiki/Server-Win-Mac#recording (HTTP 200 but missing anchor)                                                                                             
- (391:248) 'More inf..' => http://127.0.0.1:4000/es/wiki/Compiling#the-headless-build-flag (HTTP 200 but missing anchor)                                                                                   

http://127.0.0.1:4000/es/wiki/Server-Rpi
- (195:304) 'servidor..' => http://127.0.0.1:4000/es/wiki/Server-Linux#ejecutar-un-servidor-headless (HTTP 200 but missing anchor)                                                                          
- (198:62) 'estas in..' => http://127.0.0.1:4000/es/wiki/Server-Linux#compila-las-fuentes-crea-un-usuario (HTTP 200 but missing anchor)                                                                     
- (210:174) 'guía par..' => http://127.0.0.1:4000/es/wiki/Server-Linux#crea-un-script-de-arranque (HTTP 200 but missing anchor)                                                                             

http://127.0.0.1:4000/es/wiki/Server-Win-Mac
- (193:57) 'instrucc..' => http://127.0.0.1:4000/es/wiki/Server-Linux#ejecutar-el-servidor-con-interfaz-gr%C3%A1fica (HTTP 200 but missing anchor)                                                          
- (227:72) 'siguiend..' => http://127.0.0.1:4000/es/wiki/Server-Linux#controlar-las-grabaciones (HTTP 200 but missing anchor)                                                                               

http://127.0.0.1:4000/fr/wiki/Client-Troubleshooting
- (238:118) 'page de ..' => http://127.0.0.1:4000/fr/wiki/Server-Troubleshooting#personne-ne-peut-se-connecter-%C3%A0-mon-serveur-mais-je-peux-my-connecter-localement (HTTP 200 but missing anchor)        

http://127.0.0.1:4000/fr/wiki/Command-Line-Options
- (349:35) 'Trucs et..' => http://127.0.0.1:4000/fr/wiki/Tips-Tricks-More#Using-ctrlmidich-for-MIDI-controllers (HTTP 200 but missing anchor)                                                               

http://127.0.0.1:4000/fr/wiki/Getting-Started
- (203:363) 'matériel..' => http://127.0.0.1:4000/fr/wiki/Getting-Started#materiel_recommand%C3%A9 (HTTP 200 but missing anchor)                                                                            

http://127.0.0.1:4000/fr/wiki/Hardware-Setup
- (233:22) 'client i..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Linux#configure-jack-with-qjackctl (HTTP 200 but missing anchor)                                                                  

http://127.0.0.1:4000/fr/wiki/Installation-for-Linux
- (281:160) 'instruct..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Linux#compiler-jamulus (HTTP 200 but missing anchor)                                                                             
- (296:238) 'serveur ..' => http://127.0.0.1:4000/fr/wiki/Server-Linux#serveur-sans-iug (HTTP 200 but missing anchor)                                                                                       

http://127.0.0.1:4000/fr/wiki/Installation-for-Windows
- (193:178) 'défilez ..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Windows#setting-up-asio4all (HTTP 200 but missing anchor)                                                                        

http://127.0.0.1:4000/fr/wiki/Onboarding
- (196:194) 'ASIO4All..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Windows#setting-up-asio4all (HTTP 200 but missing anchor)                                                                        

http://127.0.0.1:4000/fr/wiki/Running-a-Server
- (209:241) 'Rule Num..' => http://127.0.0.1:4000/fr/wiki/Getting-Started#having-trouble-cant-keep-in-time (HTTP 200 but missing anchor)                                                                    
- (211:372) 'follow R..' => http://127.0.0.1:4000/fr/wiki/Getting-Started#having-trouble-cant-keep-in-time (HTTP 200 but missing anchor)                                                                    

http://127.0.0.1:4000/fr/wiki/Server-Linux
- (215:6) 'Get the ..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)                                                                            
- (215:92) 'dependen..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)                                                                          

http://127.0.0.1:4000/fr/wiki/Server-Rpi
- (199:7) 'Get the ..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)                                                                            
- (199:93) 'dependen..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)                                                                          
- (202:155) 'default ..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Linux#compile-this-bad-boy (HTTP 200 but missing anchor)                                                                         

http://127.0.0.1:4000/it/wiki/Client-Troubleshooting
- (211:101) 'ASIO4All..' => http://127.0.0.1:4000/it/wiki/Installation-for-Windows#setting-up-asio4all (HTTP 200 but missing anchor)                                                                        

http://127.0.0.1:4000/it/wiki/Command-Line-Options
- (256:32) 'See note' => http://127.0.0.1:4000/it/wiki/Choosing-a-Server-Type#3-central (HTTP 200 but missing anchor)                                                                                       
- (370:28) 'Tips & T..' => http://127.0.0.1:4000/it/wiki/Tips-Tricks-More#Using-ctrlmidich-for-MIDI-controllers (HTTP 200 but missing anchor)                                                               

http://127.0.0.1:4000/it/wiki/Hardware-Setup
- (254:22) 'client i..' => http://127.0.0.1:4000/it/wiki/Installation-for-Linux#configure-jack-with-qjackctl (HTTP 200 but missing anchor)                                                                  

http://127.0.0.1:4000/it/wiki/Installation-for-Linux
- (305:47) 'ottenere..' => http://127.0.0.1:4000/it/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)                                                                           
- (305:135) 'istruzio..' => http://127.0.0.1:4000/it/wiki/Installation-for-Linux#compile-this-bad-boy (HTTP 200 but missing anchor)                                                                         

http://127.0.0.1:4000/it/wiki/Onboarding
- (217:194) 'ASIO4All..' => http://127.0.0.1:4000/it/wiki/Installation-for-Windows#setting-up-asio4all (HTTP 200 but missing anchor)                                                                        

http://127.0.0.1:4000/it/wiki/Running-a-Server
- (230:241) 'Rule Num..' => http://127.0.0.1:4000/it/wiki/Getting-Started#having-trouble-cant-keep-in-time (HTTP 200 but missing anchor)                                                                    
- (232:372) 'follow R..' => http://127.0.0.1:4000/it/wiki/Getting-Started#having-trouble-cant-keep-in-time (HTTP 200 but missing anchor)                                                                    

http://127.0.0.1:4000/it/wiki/Server-Linux
- (236:6) 'Get the ..' => http://127.0.0.1:4000/it/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)                                                                            
- (236:92) 'dependen..' => http://127.0.0.1:4000/it/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)                                                                          

http://127.0.0.1:4000/it/wiki/Server-Rpi
- (220:7) 'Get the ..' => http://127.0.0.1:4000/it/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)                                                                            
- (220:93) 'dependen..' => http://127.0.0.1:4000/it/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)                                                                          
- (223:155) 'default ..' => http://127.0.0.1:4000/it/wiki/Installation-for-Linux#compile-this-bad-boy (HTTP 200 but missing anchor)                                                                         

http://127.0.0.1:4000/it/wiki/Server-Win-Mac
- (260:99) 'private ..' => http://127.0.0.1:4000/it/wiki/Choosing-a-Server-Type#3-central (HTTP 200 but missing anchor)                                                                                     

http://127.0.0.1:4000/wiki/Command-Line-Options
- (349:28) 'Tips & T..' => http://127.0.0.1:4000/wiki/Tips-Tricks-More#Using-ctrlmidich-for-MIDI-controllers (HTTP 200 but missing anchor)                                                                  

Warnings. Checked 10175 links, 227 destination URLs (434 ignored), 55 have warnings.

File: _data/es/navigation.yml (hunk 1 above)
Comment (ignotus666) [reply _data/es/navigation.yml:12]: Done.

**Comment by trebmuh** [comment]
> http://127.0.0.1:4000/fr/wiki/Onboarding
> 
>     * (196:194) 'ASIO4All..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Windows#setting-up-asio4all (HTTP 200 but missing anchor)

This is strange, because I've put the _frenchised_ link ( **_#configuration-de-asio4all_** instead of **_#setting-up-asio4all_** ) while translating this page. See https://github.com/gegeweb/jamuluswebsite/blob/f5876024aa244a1ea1e56cd9cb2db4ceb702f225/wiki/fr/fr-Onboarding.md

**Comment by ignotus666** [comment]
Some of the broken links I can see what's happening, but others I just can't see why they fail. When adding # to link to a specific paragraph, does the spelling of the words have to match exactly including capital letters? I think I recall hyphens creating problems - would it be best to get rid of them? Not being able to test all the links before it goes live forces us to make blind guesses sometimes... Maybe we could go ahead and merge translate into release, and then we can each work on fixing the links, with the ability to see if they are actually fixed?

File: _data/es/navigation.yml (hunk 1 above)
Comment (ann0see) [reply _data/es/navigation.yml:12]: Great

**Comment by ann0see** [comment]
> Not being able to test all the links before it goes live forces us to make blind guesses sometimes

You can build the page locally if you have setup Jekyll. But I'll merge this now. 

But there are a few issues online now. Will look at the German translation tomorrow.

**Comment by trebmuh** [comment]
http://127.0.0.1:4000/fr/wiki/Client-Troubleshooting
- (238:118) 'page de ..' => http://127.0.0.1:4000/fr/wiki/Server-Troubleshooting#personne-ne-peut-se-connecter-%C3%A0-mon-serveur-mais-je-peux-my-connecter-localement (HTTP 200 but missing anchor)        
**trebmuh:** the targeted page isn't translated in French yet. Better not to do anything yet; and rechecked when it'll be translated.

http://127.0.0.1:4000/fr/wiki/Command-Line-Options
- (349:35) 'Trucs et..' => http://127.0.0.1:4000/fr/wiki/Tips-Tricks-More#Using-ctrlmidich-for-MIDI-controllers (HTTP 200 but missing anchor)                                                               
**trebmuh:** fixed in #177 

http://127.0.0.1:4000/fr/wiki/Getting-Started
- (203:363) 'matériel..' => http://127.0.0.1:4000/fr/wiki/Getting-Started#materiel_recommand%C3%A9 (HTTP 200 but missing anchor)                  
**trebmuh:** fixed in #177 

http://127.0.0.1:4000/fr/wiki/Hardware-Setup
- (233:22) 'client i..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Linux#configure-jack-with-qjackctl (HTTP 200 but missing anchor)           
**trebmuh:** the Hardware-Setup page isn't translated in French yet. Better not to do anything yet; and rechecked when it'll be translated.

http://127.0.0.1:4000/fr/wiki/Installation-for-Linux
- (281:160) 'instruct..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Linux#compiler-jamulus (HTTP 200 but missing anchor)                                                                             
**trebmuh:** fixed in #177 
- (296:238) 'serveur ..' => http://127.0.0.1:4000/fr/wiki/Server-Linux#serveur-sans-iug (HTTP 200 but missing anchor)      
**trebmuh:** the targeted page isn't translated in French yet. Better not to do anything yet; and rechecked when it'll be translated.

http://127.0.0.1:4000/fr/wiki/Installation-for-Windows
- (193:178) 'défilez ..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Windows#setting-up-asio4all (HTTP 200 but missing anchor)  
**trebmuh:** fixed in #177 

http://127.0.0.1:4000/fr/wiki/Onboarding
- (196:194) 'ASIO4All..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Windows#setting-up-asio4all (HTTP 200 but missing anchor)          
**trebmuh:** this one is a false-positive

http://127.0.0.1:4000/fr/wiki/Running-a-Server
- (209:241) 'Rule Num..' => http://127.0.0.1:4000/fr/wiki/Getting-Started#having-trouble-cant-keep-in-time (HTTP 200 but missing anchor)                                                                    
- (211:372) 'follow R..' => http://127.0.0.1:4000/fr/wiki/Getting-Started#having-trouble-cant-keep-in-time (HTTP 200 but missing anchor)               
**trebmuh:** the Running-a-Server page isn't translated in French yet. Better not to do anything yet; and rechecked when it'll be translated.

http://127.0.0.1:4000/fr/wiki/Server-Linux
- (215:6) 'Get the ..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)                                                                            
- (215:92) 'dependen..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor) 
**trebmuh:** the Server-Linux page isn't translated in French yet. Better not to do anything yet; and rechecked when it'll be translated.

http://127.0.0.1:4000/fr/wiki/Server-Rpi
- (199:7) 'Get the ..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)                                                                            
- (199:93) 'dependen..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)                                                                          
- (202:155) 'default ..' => http://127.0.0.1:4000/fr/wiki/Installation-for-Linux#compile-this-bad-boy (HTTP 200 but missing anchor)  
**trebmuh:** the Server-Rpi page isn't translated in French yet. Better not to do anything yet; and rechecked when it'll be translated.

**Comment by ann0see** [comment]
@ignotus666 

there's now even a 404:

https://jamulus.io/de/wiki/Client-Troubleshooting
- (1:5916) 'ASIO4All..' => https://jamulus.io/de/wiki/Installation-for-Windows#setting-up-asio4all (HTTP 200 but missing anchor)                                                                            

https://jamulus.io/de/wiki/Command-Line-Options
- (3:6566) 'Tipps & ..' => https://jamulus.io/de/wiki/Tips-Tricks-More#Using-ctrlmidich-for-MIDI-controllers (HTTP 200 but missing anchor)                                                                  

https://jamulus.io/de/wiki/Hardware-Setup
- (1:8240) 'client i..' => https://jamulus.io/de/wiki/Installation-for-Linux#configure-jack-with-qjackctl (HTTP 200 but missing anchor)                                                                     

https://jamulus.io/de/wiki/Server-Linux
- (1:6917) 'Get the ..' => https://jamulus.io/de/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)                                                                              
- (1:7003) 'dependen..' => https://jamulus.io/de/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)                                                                             
- (62:2672) 'More inf..' => https://jamulus.io/de/wiki/Compiling#the-headless-build-flag (HTTP 200 but missing anchor)                                                                                      

https://jamulus.io/de/wiki/Server-Rpi
- (1:5847) 'Get the ..' => https://jamulus.io/de/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)                                                                              
- (1:5933) 'dependen..' => https://jamulus.io/de/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)                                                                             
- (1:6416) 'default ..' => https://jamulus.io/de/wiki/Installation-for-Linux#compile-this-bad-boy (HTTP 200 but missing anchor)                                                                             

https://jamulus.io/de/wiki/Server-Win-Mac
- (1:8416) 'private ..' => https://jamulus.io/de/wiki/Choosing-a-Server-Type#3-central (HTTP 200 but missing anchor)                                                                                        

https://jamulus.io/es/wiki/Server-Linux
- (1:7103) 'Ver esta..' => https://jamulus.io/es/wiki/Jamulus-Cloud (HTTP 404)
- (1:7742) 'ver la n..' => https://jamulus.io/es/wiki/Server-Linux#el-indicador-de-compilaci%C3%B3n-headless (HTTP 200 but missing anchor)                                                                  
- (48:818) 'instrucc..' => https://jamulus.io/es/wiki/Server-Linux#comp%C3%ADla-las-fuentes-crea-un-usuario (HTTP 200 but missing anchor)                                                                   

https://jamulus.io/fr/wiki/Client-Troubleshooting
- (1:13340) 'page de ..' => https://jamulus.io/fr/wiki/Server-Troubleshooting#personne-ne-peut-se-connecter-%C3%A0-mon-serveur-mais-je-peux-my-connecter-localement (HTTP 200 but missing anchor)           

https://jamulus.io/fr/wiki/Compiling
- (1:5184) 'compiler..' => https://jamulus.io/fr/wiki/Server-Linux#serveur-sans-iug (HTTP 200 but missing anchor)                                                                                           
- (18:2935) 'serveur ..' => https://jamulus.io/fr/wiki/Server-Linux#serveur-sans-iug (HTTP 200 but missing anchor)                                                                                          

https://jamulus.io/fr/wiki/Hardware-Setup
- (1:8228) 'client i..' => https://jamulus.io/fr/wiki/Installation-for-Linux#configure-jack-with-qjackctl (HTTP 200 but missing anchor)                                                                     

https://jamulus.io/fr/wiki/Installation-for-Linux
- (11:2595) 'Obtenir ..' => https://jamulus.io/fr/wiki/Installation-for-Linux#Installation-for-Linux%23obtenir-les-sources-de-jamulus (HTTP 200 but missing anchor)                                         
- (11:3939) 'serveur ..' => https://jamulus.io/fr/wiki/Server-Linux#serveur-sans-iug (HTTP 200 but missing anchor)                                                                                          

https://jamulus.io/fr/wiki/Running-a-Server
- (1:6414) 'Rule Num..' => https://jamulus.io/fr/wiki/Getting-Started#having-trouble-cant-keep-in-time (HTTP 200 but missing anchor)                                                                        
- (1:6994) 'follow R..' => https://jamulus.io/fr/wiki/Getting-Started#having-trouble-cant-keep-in-time (HTTP 200 but missing anchor)                                                                        

https://jamulus.io/fr/wiki/Server-Linux
- (1:6905) 'Get the ..' => https://jamulus.io/fr/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)                                                                              
- (1:6991) 'dependen..' => https://jamulus.io/fr/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)                                                                             
- (62:2672) 'More inf..' => https://jamulus.io/fr/wiki/Compiling#the-headless-build-flag (HTTP 200 but missing anchor)                                                                                      

https://jamulus.io/fr/wiki/Server-Rpi
- (1:5835) 'Get the ..' => https://jamulus.io/fr/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)                                                                              
- (1:5921) 'dependen..' => https://jamulus.io/fr/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)                                                                             
- (1:6404) 'default ..' => https://jamulus.io/fr/wiki/Installation-for-Linux#compile-this-bad-boy (HTTP 200 but missing anchor)                                                                             

https://jamulus.io/it/wiki/Client-Troubleshooting
- (1:6148) 'ASIO4All..' => https://jamulus.io/it/wiki/Installation-for-Windows#setting-up-asio4all (HTTP 200 but missing anchor)                                                                            

https://jamulus.io/it/wiki/Command-Line-Options
- (3:1566) 'See note' => https://jamulus.io/it/wiki/Choosing-a-Server-Type#3-central (HTTP 200 but missing anchor)                                                                                          
- (3:6371) 'Tips & T..' => https://jamulus.io/it/wiki/Tips-Tricks-More#Using-ctrlmidich-for-MIDI-controllers (HTTP 200 but missing anchor)                                                                  

https://jamulus.io/it/wiki/Hardware-Setup
- (1:8472) 'client i..' => https://jamulus.io/it/wiki/Installation-for-Linux#configure-jack-with-qjackctl (HTTP 200 but missing anchor)                                                                     

https://jamulus.io/it/wiki/Installation-for-Linux
- (12:2585) 'ottenere..' => https://jamulus.io/it/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)                                                                             
- (12:2673) 'istruzio..' => https://jamulus.io/it/wiki/Installation-for-Linux#compile-this-bad-boy (HTTP 200 but missing anchor)                                                                            

https://jamulus.io/it/wiki/Onboarding
- (1:5939) 'ASIO4All..' => https://jamulus.io/it/wiki/Installation-for-Windows#setting-up-asio4all (HTTP 200 but missing anchor)                                                                            

https://jamulus.io/it/wiki/Running-a-Server
- (1:6658) 'Rule Num..' => https://jamulus.io/it/wiki/Getting-Started#having-trouble-cant-keep-in-time (HTTP 200 but missing anchor)                                                                        
- (1:7238) 'follow R..' => https://jamulus.io/it/wiki/Getting-Started#having-trouble-cant-keep-in-time (HTTP 200 but missing anchor)                                                                        

https://jamulus.io/it/wiki/Server-Linux
- (1:7149) 'Get the ..' => https://jamulus.io/it/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)                                                                              
- (1:7235) 'dependen..' => https://jamulus.io/it/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)                                                                             

https://jamulus.io/it/wiki/Server-Rpi
- (1:6079) 'Get the ..' => https://jamulus.io/it/wiki/Installation-for-Linux#get-jamulus-sources (HTTP 200 but missing anchor)                                                                              
- (1:6165) 'dependen..' => https://jamulus.io/it/wiki/Installation-for-Linux#install-dependencies (HTTP 200 but missing anchor)                                                                             
- (1:6648) 'default ..' => https://jamulus.io/it/wiki/Installation-for-Linux#compile-this-bad-boy (HTTP 200 but missing anchor)                                                                             

https://jamulus.io/it/wiki/Server-Win-Mac
- (1:8648) 'private ..' => https://jamulus.io/it/wiki/Choosing-a-Server-Type#3-central (HTTP 200 but missing anchor)                                                                                        

https://jamulus.io/wiki/Command-Line-Options
- (3:6371) 'Tips & T..' => https://jamulus.io/wiki/Tips-Tricks-More#Using-ctrlmidich-for-MIDI-controllers (HTTP 200 but missing anchor)                                                                     

Errors. Checked 10140 links, 228 destination URLs (298 ignored), 1 has error(s), 42 have warnings.

**Comment by trebmuh** [comment]
:+1:

**Pull request merged**